src/Sylius/Bundle/CurrencyBundle/Form/Type/CurrencyType.php

```diff
@@ -30,6 +30,12 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('exchangeRate', 'number', [
                 'label' => 'sylius.form.currency.exchange_rate',
             ])
+            ->add('enabled', 'checkbox', [
```

Comment:
You have to update `CurrencyTypeSpec` too. Otherwise your build will [fail anyway](https://travis-ci.org/Sylius/Sylius/jobs/115588374#L2645)